pkg/model/proto/model.proto

```diff
@@ -734,7 +741,7 @@ message DeleteFileShareOpts {
     // The Context
     string context = 6;
     // The Serialized profile
-    string profile = 7;
+    string ProfileId = 7;
```

Comment:
Same here

Reply:
checked with the fileshare team, @PravinRanjan10 this should be ProfileId